### PR TITLE
Update Distribution Page styling

### PR DESCRIPTION
**Pull request by youngkidwarrior**
Notably this PR is missing the inset shadow on the countdown components. Inset shadow is hard to do in react native/tamagui, so it will be dedicated to a seperate PR

[Here is the figma this PR is based on](https://www.figma.com/file/k7ct2aqK1zUvZripOWtW8h/send-landing?type=design&node-id=1399-4453&mode=design&t=JRPvN6cLdoUkLS3N-4)

### CODE REVIEW

File: packages/app/features/distributions/components/distribution-stat-cards.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { DistributionClaimButton } from './DistributionClaimButton'
 
 export function DistributionsStatCard(cardProps: CardProps) {
   return (
-    <Card theme="gold" f={1} $gtMd={{ minWidth: 240, flex: 1, flexBasis: 0 }} {...cardProps}>
+    <Card bg="$background" f={1} $gtMd={{ minWidth: 240, flex: 1, flexBasis: 0 }} {...cardProps}>
```
Comment (0xBigBoss) [inline packages/app/features/distributions/components/distribution-stat-cards.tsx:15]: not sure this is the right color.

Screenshot 2024-01-06 at 12 32 21

File: packages/app/features/distributions/screen.tsx [hunk 2]
```diff
@@ -11,15 +11,14 @@ import { DistributionsTable } from './components/DistributionsTable'
 
 export function DistributionsScreen() {
   return (
-    <Container>
-      <ScrollView f={3} fb={0}>
-        <YStack gap="$6" pt="$5" pb="$8">
-          <YStack gap="$8">
-            <DistributionSection />
-          </YStack>
+    <ScrollView f={3} fb={0} bg="#F0EEE9">
```
Comment (0xBigBoss) [inline packages/app/features/distributions/screen.tsx:14]: what is this hard-coded bg color for ?

Comment (youngkidwarrior) [reply packages/app/features/distributions/screen.tsx:14]: Yeah, I wasn't going to merge till this was settled. 

This background color and it's dark mode equivalent don't exist in the theme yet, from what I can tell

Light: #F0EEE9 
Dark: #292929

File: packages/app/features/distributions/components/distribution-stat-cards.tsx (hunk 1 above)
Comment (0xBigBoss) [inline packages/app/features/distributions/components/distribution-stat-cards.tsx:15]: ignore my comment earlier here, was a problem with my local setup

**Comment by youngkidwarrior** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #49** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/49?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/49?utm_source=stack-comment).

File: packages/app/features/distributions/components/distribution-stat-cards.tsx [hunk 3]
```diff
@@ -115,14 +117,20 @@ export function DistributionTimeCard(props: CardProps) {
...
-      ? new Date()
+      ? now
       : isBeforeQualification
         ? distribution?.qualification_start
         : isDuringQualification
           ? distribution?.qualification_end
           : isClaimable
             ? distribution?.claim_end
-            : new Date()
+            : now
```
Comment (0xBigBoss) [inline packages/app/features/distributions/components/distribution-stat-cards.tsx:127]: hmmm why do the conditions start over again?

File: packages/app/features/distributions/components/distribution-stat-cards.tsx [hunk 4]
```diff
@@ -220,25 +250,39 @@ export function DistributionTimeCard(props: CardProps) {
...
+                    fontSize: '$10',
                   }}
                 >
                   {String(timeRemaining.hours).padStart(2, '0')}
-                </Paragraph>
+                </Text>
                 <Paragraph
                   f={3}
-                  mt="$2"
+                  color={'$gold9'}
```
Comment (0xBigBoss) [inline packages/app/features/distributions/components/distribution-stat-cards.tsx:285]: We should honestly CTRL+F `$gold` replace `$accent`. WDYT @youngkidwarrior ?

**Review by 0xBigBoss — APPROVED** [review]
looks good other than some minor comments.

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Jan 24, 10:08 PM**: [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/49) rebased this pull request as part of a merge.

File: packages/app/features/distributions/components/distribution-stat-cards.tsx (hunk 3 above)
Comment (youngkidwarrior) [reply packages/app/features/distributions/components/distribution-stat-cards.tsx:127]: Yeah, that's weird. This is super unreadable anyway. I'm going to change it to a switch(true) statement

**Pull request merged**